tests/manual/web/HTML5VideoControlTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.web.WebView;
+import javafx.stage.Stage;
+
+public class HTML5VideoControlTest extends Application {
+    @Override
+    public void start(Stage primaryStage) {
+        WebView webView = new WebView();
+        webView.getEngine().loadContent(
+            "<video width=\"320\" height=\"240\" controls>\n" +
+            "  <source src=\"your_video_url.mp4\" type=\"video/mp4\">\n" +
+            "  Your browser does not support the video tag.\n" +
+            "</video>"
+        );
+
+        VBox root = new VBox();
+        root.setSpacing(20);
+
+        HBox content = new HBox();
+        content.getChildren().addAll(createInstructionsBox(), webView); // Swap the order here
+
+        root.getChildren().addAll(content);
+
+        Scene scene = new Scene(root, 800, 400);
+        primaryStage.setScene(scene);
+        primaryStage.setTitle("HTML5 Video Player with Instructions");
+        primaryStage.show();
+    }
+
+    private VBox createInstructionsBox() {
+        VBox instructionsBox = new VBox();
+        instructionsBox.setStyle("-fx-background-color: #f0f0f0;");
+        instructionsBox.setPrefWidth(400);
+
+        instructionsBox.getChildren().addAll(
+            new javafx.scene.control.Label("Instructions:"),
+            new javafx.scene.control.Label("1. Click 'Play' to start the video."),
+            new javafx.scene.control.Label("2. Use the video controls to pause,"),
+            new javafx.scene.control.Label("2. controls should be visible,"),
+            new javafx.scene.control.Label("4. it works!")
```

Comment:
You have two step 2 lines, and step 4 isn't precise enough. I would recommend replacing both line 2s and line 4 with something like:

```
    2. The media controls should be visible once the video starts playing.
    3. Use the media controls to play/pause/seek the video.
```